src/database/DataTypes/index.js

```diff
@@ -16,13 +16,17 @@ export class SyncOut extends Realm.Object {}
 export class TransactionCategory extends Realm.Object {}
 export class User extends Realm.Object {}
 
-export { Item } from './Item';
+export { Abbreviation } from './Abbreviation';
```

Comment:
Just added the 3 classes here and in the schema - did an auto-alphabetical sort

Reply:
I confess that I prefer them sorted. even though it makes for a strange diff.